operate/client/src/App/ProcessInstance/TopPanel/MetadataPopover/v2/index.tsx

```diff
@@ -14,21 +14,86 @@ import {observer} from 'mobx-react';
 import {flip, offset} from '@floating-ui/react-dom';
 import {Header} from '../Header';
 import {Stack} from '@carbon/react';
-import {Details} from '../Details';
 import {Incident} from '../Incident';
 import {MultiIncidents} from '../MultiIncidents';
 import {useProcessInstance} from 'modules/queries/processInstance/useProcessInstance';
+import {useElementInstancesSearch} from 'modules/queries/elementInstances/useElementInstancesSearch';
+import {useElementInstance} from 'modules/queries/elementInstances/useElementInstance';
+import {useFlownodeInstancesStatistics} from 'modules/queries/flownodeInstancesStatistics/useFlownodeInstancesStatistics';
+import {useMemo} from 'react';
+import {Details} from './Details';
+import {createV2InstanceMetadata} from './types';
 
 type Props = {
   selectedFlowNodeRef?: SVGGraphicsElement | null;
 };
 
 const MetadataPopover = observer(({selectedFlowNodeRef}: Props) => {
   const {data: processInstance} = useProcessInstance();
-  const flowNodeId = flowNodeSelectionStore.state.selection?.flowNodeId;
+  const elementId = flowNodeSelectionStore.state.selection?.flowNodeId;
```

Comment:
There's a bug on how you're consuming the selection. You need to take into account multi instance bodies case into consideration too